base/src/expressions/lexer/test/test_ranges.rs

```diff
@@ -191,7 +224,7 @@ fn test_range_column_absolute2() {
                 row: LAST_ROW,
                 absolute_column: true,
                 absolute_row: true,
-            }
+            },
```

Comment:
I am confused, why was the linter passing on this?

Reply:
I am also curious about this, so I just did some testing.  `cargo fmt --` actually treats both as valid.  However, the integration in my IDE preferred trailing commas.  I've since changed some settings and it no longer seems to be preferring that format, so we should hopefully be fine moving forward.  I'm happy to revert the change back if you prefer.

Reply:
TIL! I was under the impression `cargo fmt` would allow one or the other. Yes, please revert those changes for now and lets make a more conscious decision on hour next call.